doc/design.md

```diff
@@ -112,4 +112,38 @@ Covers:
 
 * `req~rendering.sql.confiugrable-identifier-quoting~1`
 
+Needs: impl, utest
+
+### Exasol Dialect Specific
+
+#### Converting from 64 bit Integers to INTERVAL DAY TO SECOND
+`dsn~exasol.converting-int-to-interval-day-to-second~1`
+
+The data converter converts integers to `INTERVAL DAY TO SECOND`.
+
+Covers:
+
+* `req~integer-interval-conversion~1`
+
+Needs: impl, utest
+
+#### Parsing INTERVAL DAY TO SECOND From Strings
+`dsn~exasol.parsing-interval-day-to-second-from-strings~1`
+
+The data converter can parse `INTERVAL DAY TO SECOND` from strings in the following format:
+
+    inverval-d2s = [ days SP ] hours ":" minutes [ ":" seconds [ "." milliseconds ] ]
```

Comment:
typo: inverval-d2s --> interval-d2s
At first glance, notation is not completely clear.
Maybe concret examples for string intervals (interval-d2s, hours, minutes ...) could be helpul (just suggestion)

Reply:
Added examples. Will add introduction including ABNF mention in separate ticket.